cmd/go-tpc/tpch.go

```diff
@@ -67,6 +67,62 @@ func executeTpch(action string) {
 	w.OutputStats(true)
 }
 
+func getServerVersion(db *sql.DB) (string, error) {
+	var version string
+	err := db.QueryRow("SELECT VERSION()").Scan(&version)
+	return version, err
+}
+
+// isValidTuningVersion checks if the version is a valid TiDB version for query tuning params @queryTuningVars
+// @version the output serverVersion of tidb using @getServerVersion
+// INFO: Should be optimized if some vars are changed in the future.
+func isValidTuningVersion(version string) bool {
```

Comment:
I think we should split this function into two: one extracting a version struct of `{major int, minor int, patch string}`, the other asserting if a given system variable is supported by the given version struct (`func isSysVarSupported(version versionInfo, sysVar string)`).